gw.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: install.istio.io/v1alpha1
```

Comment:
I'm thinking this shouldn't be in the root directory. Maybe under samples?

Reply:
this should be deleted - generated from a samples script.

Reply:
If it comes from a script, should it be added to .gitignore so it's not added?

Reply:
Generated via `script.sh > gw.yaml` - I don't think users will do this. They'd actually pipe it to `manifest generate`. Probably removing that from this PR in favor of #27250